src/Cake.Npm/BumpVersion/NpmBumpVersionSettings.cs

```diff
@@ -35,6 +35,12 @@ public NpmBumpVersionSettings()
         /// </summary>
         public string Version  {get;set; }
 
+        /// <summary>
+        /// Gets or sets the --git-tag-version=false option.
+        /// Tag the commit when using the npm version command. Setting this to true results in no commit being made at all.
+        /// </summary>
+        public bool NoGitTagVersion { get; set; }
```

Comment:
To make it easier to understand for the user we should use the same name as the npm command has (`GitTagVersion`) and make it nullable: If not defined `git-tag-version` won't be added and default value as defined by the npm command is used, if set `git-tag-version` is added with the corresponding value.